gapis/api/vulkan/frame_loop.go

```diff
@@ -371,6 +377,22 @@ func (f *frameLoop) Transform(ctx context.Context, cmdId api.CmdID, cmd api.Cmd,
 			f.loopTerminated = true
 			log.D(ctx, "FrameLoop: end loop at frame %v cmdId %v, cmd is %v.", f.frameNum, cmdId, cmd)
 
+			// This command is the last in the loop so lets add it to the captured commands so we don't need to special case it.
+			f.capturedLoopCmds = append(f.capturedLoopCmds, cmd)
+			f.capturedLoopCmdIds = append(f.capturedLoopCmdIds, cmdId)
+
+			// If we are looping zero times we can just drop the commands inside the loop.
+			if f.loopCount == 0 {
```

Comment:
so `loopcount == 0` means skip all commands and  `loopcount == 1`  means no loop and just replay normally, right?

Reply:
Correct. Same as a c program.